### PR TITLE
QE: Change 5.0 BV URL for 5.0.1

### PR DESCRIPTION
**Pull request by nodeg**
Changing the container URL for the 5.0.1 BV.

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf [hunk 1]
```diff
@@ -177,7 +177,7 @@ module "server_containerized" {
   }
   main_disk_size = 3000
   runtime = "podman"
-  container_repository = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile/suse/manager/5.0/x86_64"
+  container_repository = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile/suse/manager/5.0/x86_64"
```
Comment (vandabarata) [inline terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:180]: please add a comment explaining it's temporary, and you can even link my github card to change this in the future - https://github.com/SUSE/spacewalk/issues/25120

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf [hunk 2]
```diff
@@ -215,9 +215,7 @@ module "proxy_containerized" {
     password = "admin"
   }
   runtime = "podman"
-  container_repository = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile/suse/manager/5.0/x86_64"
-  // Most recent code. Enable again once Beta 2 will be approved:
-  // container_repository = "registry.suse.de/devel/galaxy/manager/head/containerfile/suse/manager/5.0/x86_64"
+  container_repository = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile/suse/manager/5.0/x86_64"
```
Comment (vandabarata) [inline terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:218]: same here

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf (hunk 1 above)
Comment (nodeg) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:181]: Done.

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf (hunk 2 above)
Comment (nodeg) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:220]: Done.

**Review by vandabarata — APPROVED** [review]

**Pull request merged**